app/assets/javascripts/discourse/scripts/activate-account.js

```diff
@@ -1,19 +1,17 @@
 (function () {
-  setTimeout(function () {
```

Comment:
I'm not sure why were were delaying the event listener setup by 50ms. The delay was causing flakiness in the system spec, and doesn't seem to be required any more 🤷‍♂️